Fix grammar in client function comments

LeviHarrison opened the pull request:
<!--
Please make sure you've read and understood our contributing guidelines;
https://github.com/moby/moby/blob/master/CONTRIBUTING.md

** Make sure all your commits include a signature generated with `git commit -s` **

For additional information on our contributing process, read our contributing
guide https://docs.docker.com/opensource/code/

If this is a bug fix, make sure your description includes "fixes #xxxx", or
"closes #xxxx"

Please provide the following information:
-->

**- What I did**

I was looking through the client documentation and found a small grammar mistake in one of the comments, and ended up falling down this rabbit hole. I change certain words and added punctuation to the comments of functions. I didn't get everything, nor is my grammar perfect, and I'd be happy to move things around.

**- Description for the changelog**
<!--
Write a short (one line) summary that describes the changes in this
pull request for inclusion in the changelog:
-->

Grammar fixes in exported client functions

**- A picture of a cute animal (not mandatory but encouraged)**

![image](https://user-images.githubusercontent.com/54278938/107890945-c535ba00-6ee9-11eb-9a83-bd4fbb4ec610.png)

thaJeztah commented on client/image_build.go:
```diff
@@ -14,8 +14,8 @@ import (
-// ImageBuild sends request to the daemon to build images.
-// The Body in the response implement an io.ReadCloser and it's up to the caller to
+// ImageBuild sends a request to the daemon to build images.
+// The body in the response implements an io.ReadCloser and it's up to the caller to
```
I think in this case, `Body` refers to the `Body` field in the response; https://github.com/moby/moby/blob/46cdcd206c56172b95ba5c77b827a722dab426c5/client/image_build.go#L42-L45

thaJeztah reviewed (changes requested):
thanks! left some comments

LeviHarrison commented:
Ok I removed all code changes that were just adding a period (I did leave one in that removed a double period) and recapitalized `Body`.

thaJeztah commented:
Thanks! Could you perhaps squash the commits? (let me know if you need help/instructions doing so 🤗)

LeviHarrison committed:
Fix grammar in client function comments

Changes certain words and adds punctuation to the comments of functions in the client package, which end up in the GoDoc documentation. Areas where only periods were needed were ignored to prevent excessive code churn.

Signed-off-by: Levi Harrison <levisamuelharrison@gmail.com>

LeviHarrison commented:
A squash has (sort of) been performed. Ignore all the force pushes...

thaJeztah reviewed (approved):
awesome!

LGTM

tianon reviewed (approved):
LGTM :+1:
